src/Hintman/Core/Hint.hs

```diff
@@ -53,6 +54,16 @@ parseHintType :: Text -> Maybe HintType
 parseHintType = inverseMap showHintType
 {-# INLINE parseHintType #-}
 
+-- | Creates a simple suggestion 'Hint' from provided type, title and body (note is omitted).
+simpleSuggestion :: HintType -> Text -> Text -> Hint
+simpleSuggestion hintType hintHeader hintBody = Hint
+    { hintHeader = hintHeader
+    , hintBody = hintBody
+    , hintIsSuggestion = True
+    , hintNote = ""
+    , hintType = hintType
+    }
```

Comment:
As `RecordWildCards` is on by default, you can use the variables from the scope this way:

```haskell
simpleSuggestion hintType hintHeader hintBody = Hint
    { hintIsSuggestion = True
    , hintNote = ""
    , ...
    }
```